docs/formatV3.txt

```diff
@@ -628,6 +635,35 @@ files will impact performance and synchronization times, and could make the data
 inaccessible on mobile platforms. It is advised to keep the total database size
 below 100MB.
 
+[31] The passkey parameters fields are listed below. If an entry contains
+a passkey, all fields must be present and non-empty. Below are references
+to the official specification documenting each field, including requirements
+of the contents.
+
+    Passkey Credential ID
+    https://www.w3.org/TR/webauthn-2/#credential-id
+
+    Passkey Relying Party ID
+    https://www.w3.org/TR/webauthn-2/#relying-party-identifier
+
+    Passkey User Handle
+    https://www.w3.org/TR/webauthn-2/#user-handle
+
+    Passkey Algorithm ID = IANA COSE Algorithm Identifier
+    https://www.w3.org/TR/webauthn-2/#sctn-alg-identifier
+    Encode as signed (!) 32 bit little-endian value, example for ES256:
+    0xf9 0xff 0xff 0xff
+
+    Passkey Private Key
```

Comment:
Since we will not export the private key, I see no advantage in keeping it in PEM format. DER is more compact and just as universal.

Reply:
Thanks, good suggestion, I've updated the PR.